src/makeRobot.js

```diff
@@ -38,7 +38,58 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
```

Comment:
add spaces between object methods
```suggestion
    },
    
    get info() {
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
    },
    
    get location() {
      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
```